### ISSUE TITLE
Cannot remove a mirror

### ISSUE THREAD

**Issue by eschnett** [LLNL/spack#319]
According to the documentation, you can remove a mirror like this:
```
$ spack mirror remove -h
usage: spack mirror remove [-h] name

positional arguments:
  name

optional arguments:
  -h, --help  show this help message and exit
```

However:
```
$ spack mirror remove cmake
==> Error: Unknown scope None.  Valid options are site, user
```

**Comment by tgamblin** [comment]
This will be fixed as part of #210.

**Comment by tgamblin** [comment]
Should be fixed by e8e6368, part of #210.  See if it works for you.
